srcpkgs/xbps-triggers/files/gtk-icon-cache

```diff
@@ -23,8 +23,8 @@ targets)
 run)
 	if [ "$TARGET" = "post-remove" -a "${PKGNAME}" = "gtk+" ]; then
 		for dir in ${gtk_iconcache_dirs}; do
-			if [ -f ${dir}/icon-theme.cache ]; then
-				rm -f ${dir}/icon-theme.cache
+			if [ -f .${dir}/icon-theme.cache ]; then
+				rm -f .${dir}/icon-theme.cache
```

Comment:
Would it be better to use `./` instead of `.`?

Reply:
Why even bother with `./`?

Reply:
@sgn I believe these are always generated automatically, so we are sure that they will contain a leading slash.